src/main/java/canelo/Canelo.java

```diff
@@ -0,0 +1,314 @@
+package canelo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Canelo {
+    static Scanner input = new Scanner(System.in);
+    static int numTasks = 0;
+    static ArrayList<Task> tasks = new ArrayList<>();
+
+    private static final String FILE_PATH = "caneloData.txt";
+
+    static final String BY = "/by";
+    static final String FROM = "/from";
+    static final String TO = "/to";
+
+    static final int MINIMUM_TODO_LENGTH = 5;
+    static final int MINIMUM_MARK_LENGTH = 5;
+    static final int MINIMUM_UNMARK_LENGTH = 7;
+    static final int MINIMUM_DEADLINE_LENGTH = 9;
+    static final int DEADLINE_BY_LENGTH = 4;
+    static final int MINIMUM_EVENT_LENGTH = 6;
+    static final int EVENT_TO_LENGTH = 4;
+    static final int MINIMUM_DELETE_LENGTH = 7;
+
+    static final String LINE = "____________________________________________________________";
+
+    private static void printStartMessage() {
+        System.out.println(LINE);
+        System.out.println("""
+                 Hello! I'm Canelo :)
+                 What can I do for you?""");
+        System.out.println(LINE);
+    }
+
+    private static void printEndMessage() {
+        System.out.println(LINE);
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(LINE);
+    }
+
+    private static boolean isValidUserInput(String userInput) {
+        return !userInput.equals("bye") && !userInput.equals("q");
+    }
+
+    private static String determineTaskType(String userInput) {
```

Comment:
what is the purpose of this method? you are returning the same string that is the switch case. can you think of better ways to handle it?